### PR TITLE
8307573: Implementation of JEP 449: Deprecate the Windows 32-bit x86 Port for Removal

### PR DESCRIPTION
**Pull request by gdams**
Deprecate the Windows x86-32 port, with the intent to remove it in a future release.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8307573](https://bugs.openjdk.org/browse/JDK-8307573): Implementation of JEP 449: Deprecate the Windows 32-bit x86 Port for Removal

### Reviewers
 * [Erik Joelsson](https://openjdk.org/census#erikj) (@erikj79 - **Reviewer**) ⚠️ Review applies to [8c090a71](https://git.openjdk.org/jdk/pull/13852/files/8c090a7126fd70de653a132bb17c6161e50bd2cf)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13852/head:pull/13852` \
`$ git checkout pull/13852`

Update a local copy of the PR: \
`$ git checkout pull/13852` \
`$ git pull https://git.openjdk.org/jdk.git pull/13852/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13852`

View PR using the GUI difftool: \
`$ git pr show -t 13852`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13852.diff">https://git.openjdk.org/jdk/pull/13852.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13852#issuecomment-1537104017)

### CODE REVIEW

File: doc/building.md [hunk 1]
```diff
@@ -126,6 +126,8 @@ space is required.
 Even for 32-bit builds, it is recommended to use a 64-bit build machine, and
 instead create a 32-bit target using `--with-target-bits=32`.
 
+Note: The Widows 32-bit x86 port is deprecated.
```
Comment (djelinski) [inline doc/building.md:129]:
```suggestion
Note: The Windows 32-bit x86 port is deprecated.
```

File: doc/building.md [hunk 2]
```diff
@@ -199,6 +201,8 @@ rule also applies to input to the build system, e.g. in arguments to
 `--with-msvcr-dll=c:\msvcr100.dll`. For details on this conversion, see the section
 on [Fixpath](#fixpath).
 
+Note: The Widows x32-bit x86 port is deprecated.
```
Comment (djelinski) [inline doc/building.md:204]:
```suggestion
Note: The Windows x32-bit x86 port is deprecated.
```

File: doc/building.md (hunk 1 above)
Comment (gdams) [reply doc/building.md]: updated PTAL

File: doc/building.html [hunk 3]
```diff
@@ -323,6 +323,7 @@ <h3 id="building-on-x86">Building on x86</h3>
 <p>Even for 32-bit builds, it is recommended to use a 64-bit build
 machine, and instead create a 32-bit target using
 <code>--with-target-bits=32</code>.</p>
+<p>Note: The Windows 32-bit x86 port is deprecated.</p>
```
Comment (dholmes-ora) [inline doc/building.html:326]: Suggestion:
> The Windows 32-bit x86 port is deprecated and may be removed in a future release.

**Review by erikj79 — APPROVED** [review]

**Comment by gdams** [comment]
This JEP has now been moved from "Proposed to target" to "Targeted" which means I think we can merge this now.

**Comment by gdams** [comment]
> @gdams please see my comment above re the wording. Thanks

@dholmes-ora updated PTAL

**Review by dholmes-ora — COMMENTED** [review]
Thanks for the update

**Pull request closed without merging**